backend/config/default.json

```diff
@@ -1,5 +1,4 @@
 {
   "ip": "127.0.0.1",
-  "port": 8082,
-  "csafValidatorLibMock": false
```

Comment:
jooc: Why do we remove the `"csafValidatorLibMock": false`?

Reply:
This option is not needed. It was a noop.